docs/ref/contrib/gis/install/postgis.txt

```diff
@@ -7,20 +7,25 @@ into a spatial database. :ref:`geosbuild`, :ref:`proj4` and
 :ref:`gdalbuild` should be installed prior to building PostGIS. You
 might also need additional libraries, see `PostGIS requirements`_.
 
-The `psycopg2`_ module is required for use as the database adapter when using
-GeoDjango with PostGIS.
+The `psycopg`_ or `psycopg2`_ module is required for use as the database
+adapter when using GeoDjango with PostGIS.
 
 On Debian/Ubuntu, you are advised to install the following packages:
 ``postgresql-x.x``, ``postgresql-x.x-postgis``, ``postgresql-server-dev-x.x``,
```

Comment:
```suggestion
``postgresql-x``, ``postgresql-x-postgis-3``, ``postgresql-server-dev-x``,
```
PostgreSQL 12 is the minimum version supported by Django 4.2. Since PostgreSQL 10 the version number is incremented by 1 every time, so I have updated the package names with the really available ones in the Debian/Ubuntu repositories.
PS. there is no `postgresql-x-postgis-2.x` package.

Reply:
This is a cleanup in docs that can be backported. Please submit a separate PR with this change.

Reply:
I submitted the PR #16385